_data/tokens/spacing.yml

```diff
@@ -7,6 +7,7 @@ positive:
   small:
     - token: '05'
       value: 4px
+      conversion: 0.5
```

Comment:
Created an explicit equivalent number value here because it could not recognize the `'05'` token when assigning `converted` in `is_number.html`